edge-util/src/Microsoft.Azure.Devices.Edge.Util/CertificateHelper.cs

```diff
@@ -289,6 +323,10 @@ internal static (X509Certificate2, IEnumerable<X509Certificate2>) ParseCertifica
                 {
                     chain.Add(new X509CertificateEntry(x509Cert));
                 }
+                if (certObject is AsymmetricCipherKeyPair)
+                {
+                    certObject = ((AsymmetricCipherKeyPair)certObject).Private;
```

Comment:
Curious - what is this change for? Can you please add a comment?

Reply:
Sure I can add a comment, I am not exactly sure why this occurred but when testing with PFX certificates generated via openssl app (not dotnet core) the certObject was of type AsymmetricCipherKeyPair and thus its private key needed to accessed differently.